### DISCUSSION TITLE
Battery shows and X and no longer recognized.

### DISCUSSION THREAD

**Discussion by gabzlora** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#681] [General]
After running a calibration my mac no longer recognizes my battery.

I tried resetting SMC, NVRAM, PRAM uninstalling aldente, helper etc and no luck.

im on a MBP 15 MId 2015.

Brand new battery as of last month with no issues then.

**Comment by MatthiasKerbl** [comment]
Hi @gabzlora,

I am sorry you are having problems with your battery. I have not heard about this issue before and I do not know if it is related to AlDente.
Did you get your battery changed by Apple?

**Comment by gabzlora** [comment]
It was replaced by an authorized repair center. I would think the battery was defective. But  the problem only started during and after running the calibration process and a month after the battery was replaced and working completely fine which leads me to believe ALDENTE had something to do. Too much of a coincidence.

What files does aldente modify for managing the battery? Perhaps by replacing the files with the factory ones i can resolve my problem. I can uninstall aldente but the app hangs when I click to remove the helper. I am using the pro version from SETAPP. 

Any additional help is welcomed

**Reply by MatthiasKerbl** [reply]
If you want to remove AlDente completely from your system, you can follow the manual uninstallation guide here: https://apphousekitchen.com/how-to-uninstall-aldente-pro/
When you are doing an SMC reset as well, everything from AlDente is removed.
